### PR TITLE
[RFR] init project

### PR DESCRIPTION
**Pull request by mchaffotte**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.PHONY: test
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+start: ## Start the server
```
Comment (zyhou) [inline makefile]: Where is the install ?

File: quoridor-app/App.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function App() {
```
Comment (zyhou) [inline quoridor-app/App.js]: Use the functional way

Comment (zyhou) [reply quoridor-app/App.js]: `const App = () => {}`

File: makefile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.PHONY: test
```
Comment (zyhou) [inline makefile]: Test ?  Add other action command

File: makefile (hunk 1 above)
Comment (mchaffotte) [reply makefile]: nothing to install

File: quoridor-app/App.js (hunk 2 above)
Comment (mchaffotte) [reply quoridor-app/App.js]: it is the template of expo, I will change the content

File: makefile (hunk 3 above)
Comment (mchaffotte) [reply makefile]: I will add test

Comment (alexisjanvier) [reply makefile]: When ?

**Comment by mchaffotte** [comment]
need review

File: README.md [hunk 4]
```diff
@@ -1 +1,11 @@
-# quoridor-react-native
\ No newline at end of file
+# Quoridor
+
+Quoridor client in React Native
+
```
Comment (alexisjanvier) [inline README.md]: Missing install insttructions

Comment (alexisjanvier) [inline README.md]: Missing install instructions

File: quoridor-app/package.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "eject": "expo eject"
+  },
+  "dependencies": {
+    "expo": "^33.0.0",
+    "react": "16.8.3",
```
Comment (alexisjanvier) [inline quoridor-app/package.json]: Why is the react version fixed, unlike other dependencies?

File: README.md (hunk 4 above)
Comment (mchaffotte) [reply README.md]: It should be but it is now out of scope since the daily

File: quoridor-app/package.json (hunk 5 above)
Comment (mchaffotte) [reply quoridor-app/package.json]: expo init did it. I change them in my next feature

Comment (alexisjanvier) [reply quoridor-app/package.json]: What is done is no longer to be done ^^

File: README.md (hunk 4 above)
Comment (alexisjanvier) [reply README.md]: ok

Comment (alexisjanvier) [reply README.md]: `install instructions` i mean. Ok for docker :+1:

**Pull request merged**